src/server/migrations/20200602194903_users.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('users', (table) => {
+    table.increments();
+    table.string('uid').notNullable();
+    table.string('name').notNullable();
+    table.string('email').notNullable();
```

Comment:
Hello there. According to the DB diagram the email should be unique. For example: 
  table.string('email').unique().notNullable();